### PR TITLE
issue #48 : mount keys as volume and update readme

### LINKED ISSUE #48: Avoid Storing Keys Inside Docker Image
### Description
The current Dockerfile copies the `keys/` folder into the image, which poses a security issue.

#### 1. Mounted Volumes
Instead of embedding keys in the image, a volume can be mounted from the host machine into the Docker container at runtime using the `docker run` command.

#### 2. Cloud-Specific Solutions
For environments hosted on cloud services, secret management services offered by the cloud provider can be utilized for secure key management.

### PR DESCRIPTION
**Pull request by k-allagbe**
#48
issue48-keys-as-volumes

To avoid including secrets inside the docker image, mounted them as volume.

Closes #48

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -25,3 +25,8 @@ requirements.txt
 
 # Ignore bash scripts
 *.sh
+
+# Ignore secrets
+secrets.txt
```
Comment (rngadam) [inline .dockerignore:30]: what is secrets.txt? I don't think I see it referenced elswhere in your scripts?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: .dockerignore (hunk 1 above)
Comment (k-allagbe) [reply .dockerignore]: fixed.

**Review by rngadam — APPROVED** [review]

File: .dockerignore (hunk 1 above)
Comment (rngadam) [reply .dockerignore]: ok, would have liked to know what it was though

Comment (k-allagbe) [reply .dockerignore]: I copied from .gitignore initially. The initial creator of the .gitignore might have the answer.

**Pull request merged**